styles.css

```diff
@@ -0,0 +1,369 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap");
+body {
+  font-family: "Poppins", sans-serif;
+  padding: 0rem 1rem;
+}
+.navbar {
+  width: 99%;
+  padding: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: relative;
+  background-color: peachpuff;
```

Comment:
make variables for color and then use the variables here. So if you change the color in the variable it will change everywhere